pkg/generic/duplicate.go

```diff
@@ -3,7 +3,7 @@ package generic
 import "github.com/uvio-network/apiserver/pkg/object/objectid"
 
 // Duplicate expresses whether the given list contains duplicated items.
-func Duplicate[T string | objectid.ID | int64](lis []T) bool {
+func Duplicate[T string | objectid.ID | int | int64](lis []T) bool {
```

Comment:
I extended the generic interface here because I used `Duplicate` in the unit tests to verify that the random indices are in fact unique.